src/fragments/lib/graphqlapi/ios/subscribe-data.mdx

```diff
@@ -27,28 +27,33 @@ To listen to creation updates, you can use the following code sample:
 
 <BlockSwitcher>
 
-<Block name="Listener (iOS 11+)">
+<Block name="Async/Await">
 
 ```swift
 func createSubscription() {
-    subscription = Amplify.API.subscribe(request: .subscription(of: Todo.self, type: .onCreate), valueListener: { (subscriptionEvent) in
-        switch subscriptionEvent {
-        case .connection(let subscriptionConnectionState):
-            print("Subscription connect state is \(subscriptionConnectionState)")
-        case .data(let result):
-            switch result {
-            case .success(let createdTodo):
-                print("Successfully got todo from subscription: \(createdTodo)")
-            case .failure(let error):
-                print("Got failed result with \(error.errorDescription)")
+    var subscription: AmplifyAsyncThrowingSequence<GraphQLSubscriptionEvent<Todo>>?
+    subscription = Amplify.API.subscribe(request: .subscription(of: Todo.self, type: .onCreate))
+    guard let subscription = subscription else {
+        print("Error getting subscription")
+        return
+    }
```

Comment:
i don't think this will ever fail to get the sequence
```
let sequence = Amplify.API.subscribe(request: .subscription(of: Todo.self, type: .onCreate))

```